gh-docs/Admin_Services.md

```diff
@@ -0,0 +1,57 @@
+# How to create admin-services
+Admin Services generally follow the following steps. Here, the Message Processor has been considered as an example
+
+### Creation Flow
+
+1. Define your Admin Service function
+2. Generate the stub using WSDL
+3. Define function in MessageProcessorAdminServiceClient to call Admin Service function
```

Comment:
```suggestion
3. Define the function in `MessageProcessorAdminServiceClient` to call the Admin Service function
```

Shall we follow a convention of having all class names, file names, method names, etc. between single backticks? This is to highlight this to readers.

Reply:
I will make that change.